doc/nrf/known_issues.rst

```diff
@@ -2026,7 +2030,7 @@ SoftDevice Controller
 
 In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.
 
-.. rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0
+.. rst-class:: v2-2-0 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0
```

Comment:
Some already fixed issues of softdevice controller should be added here: 

- "rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0

DRGN-17777: Wrongly accepts LL_PAUSE_ENC_REQ packet received on an unencrypted link
   The controller should not accept an LL_PAUSE_ENC_REQ packet received on an unencrypted link"

- "rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0

DRGN-17776: Wrongly accepts the connSupervisionTimeout value set to 0
   The controller should not accept CONNECT_IND, AUX_CONNECT_REQ and CONNECTION_UPDATE_REQ packets with the connSupervisionTimeout value set to 0."

- "rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0

DRGN-19744: Assert if sync timeout short than the periodic advertising interval
   The controller would assert if trying to sync to a periodic advertiser with a sync timeout shorter than the periodic advertising interval."

- "rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0

DRGN-18089: Wrongly erase the previous periodic advertising reports
   When creating a periodic sync could in some cases erase periodic advertising reports for previously created syncs."

Reply:
@laxiLang , about DRGN-17777 and DRGN-17851: Would you have more details about these issues? Right now, their descriptions just repeat the title.

Reply:
@greg-fer hope the updated comment looks bettere

Reply:
@laxiLang , I see only an update to 17777, the 17851 description is still repeating the title.

Reply:
@greg-fer we have no better title, but the description is correct. This issue is quite general.